packages/contracts/lib.esm/index.d.ts

```diff
@@ -22,19 +22,18 @@ export interface CallOverrides extends PayableOverrides {
     from?: string | Promise<string>;
 }
 export interface PopulatedTransaction {
-    to?: string;
-    from?: string;
-    nonce?: number;
+    to?: AccountLike;
+    from?: AccountLike;
     gasLimit?: BigNumber;
-    gasPrice?: BigNumber;
     data?: string;
     value?: BigNumber;
     chainId?: number;
     type?: number;
-    accessList?: AccessList;
-    maxFeePerGas?: BigNumber;
-    maxPriorityFeePerGas?: BigNumber;
+    accessList?: AccessListish;
```

Comment:
Why do we need to change the access list to accesslistish and the same is applicable to bignumberish?